packages/concatjs/web_test/test/karma/coverage_source.js

```diff
@@ -0,0 +1,21 @@
+(function(factory) {
```

Comment:
I still don't understand, why is generated code getting checked in here?

Reply:
I compared this with the other test files in this directory, which are pretty much the same.
Should this be done differently?

Reply:
ah, now I see, there are two different directories - test/karma contains .js files and test/karma_typescript has .ts files